test/unit/org/apache/cassandra/guardrails/GuardrailConsistencyTest.java

```diff
@@ -149,6 +149,22 @@ public void testInsertWithAllowedConsistency()
         lwtInsert(ConsistencyLevel.LOCAL_QUORUM, null);
     }
 
+    private void select(ConsistencyLevel cl)
+    {
+        executeWithConsistency("SELECT * FROM %s WHERE k=1", cl, null);
+    }
+
+    @Test
+    public void testSelectWithDisallowedWriteConsistency()
+    {
+        // Select should not be affected by any disallowed write consistency levels
+        select(ConsistencyLevel.ONE);
+        select(ConsistencyLevel.LOCAL_ONE);
+        select(ConsistencyLevel.QUORUM);
+        select(ConsistencyLevel.EACH_QUORUM);
```

Comment:
`EACH_QUORUM` CL is supported for writes in a multi-region Astra DB

Reply:
Here it's just in the context of this test class that is explicitly setting some disallowed CL levels, and this test method is just ensuring that none of them apply to SELECT statements. It's not strictly necessary to even check more than one CL here.